README.md

```diff
@@ -10,7 +10,7 @@ Inspired by AppVeyor, Hangfire.Console provides a console-like logging experienc
 
 ## Features
 
- - **Provider-agnostic**: (allegedly) works with any job storage provider (currently tested with SqlServer and MongoDB). 
+ - **Provider-agnostic**: (allegedly) works with any job storage provider (currently tested with SqlServer, MemoryStorage, Redis.StackExchange and MongoDB). 
  - **100% Safe**: no Hangfire-managed data (e.g. jobs, states) is ever updated, hence there's no risk to corrupt it.
```

Comment:
I believe there were some issues with `MemoryStorage` due to incorrect implementation of Set related API. Was it fixed already?